Docker/docker-compose-hub-mssql.yml

```diff
@@ -0,0 +1,92 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# This docker compose is only intended for local testing of images that have
+# already been published to Docker Hub.
+
+version: "3.8"
+
+services:
+  db-ods:
+    image: edfialliance/ods-api-db-ods-minimal:pre-mssql
```

Comment:
Shouldn't we add the `-mssql` suffix to the image name instead of the tag name?

Reply:
we now publish the -mssql sufix as part of the tag instead of the image name, look [here](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/0ef79394cc1d785e78a7e4f74fd482a021e8b4be/Docker/Invoke-Build.ps1#L238)